index.html

```diff
@@ -0,0 +1,179 @@
+              </div>
+              <h2>2021-12-09(THU) - 12(SUN) </h2>
+              <span>@ Biel Beirut waterfront, Art center</span>
+          </div>
+      </header>
+      <main>
+          <section class="program" id="program">
+              <div class="program-h2">
+                <h2>Main Program</h2>
+                <hr class="horizontal-hr">
+              </div>
+              <div class="program-container">
+                <article class="lecture">
+                  <div class="lecture-left">
+                    <i class="fab fa-airbnb fa-3x"></i>
+                  </div>
+                  <div class="program-title">
+                    <h3>Lecture</h3>
+                  </div>
+                  <div class="lecture-right">
+                    <p>listen to speakers from various countries about the messeges of sharing and opening.</p>
+                  </div>
+                </article>
+                <article class="lecture">
+                  <div class="lecture-left">
+                    <i class="fab fa-airbnb fa-3x"></i>
+                  </div>
+                  <div class="program-title">
+                    <h3>Exhibition</h3>
+                  </div>
+                  <div class="lecture-right">
+                    <p>listen to speakers from various countries about the messeges of sharing and opening. </p>
+                  </div>
+                </article>
+                <article class="lecture">
+                   <div class="lecture-left">
+                     <i class="fab fa-airbnb fa-3x"></i>
+                   </div> 
+                   <div class="program-title">
+                     <h3>Forum</h3>
+                   </div>
+                   <div class="lecture-right">
+                     <p>listen to speakers from various countries about the messeges of sharing and opening.</p>
+                   </div>
+                 </article>
+                <article class="lecture">
+                   <div class="lecture-left">
+                     <i class="fab fa-airbnb fa-3x"></i>
+                    </div>
+                    <div class="program-title">
+                      <h3>Workshop</h3>
+                    </div>
+                    <div class="lecture-right">
+                      <p>listen to speakers from various countries about the messeges of sharing and opening.</p>
+                    </div>
+                 </article>
+                 <article class="lecture">
+                   <div class="lecture-left">
+                     <i class="fab fa-airbnb fa-3x"></i>
+                    </div>
+                    <div class="program-title">
+                      <h3>Ignite</h3>
+                    </div>
+                    <div class="lecture-right">
+                      <p>listen to speakers from various countries about the messeges of sharing and opening.</p>
+                    </div>
+                 </article>
+              </div>
```

Comment:
- [ ] ___[Optional]___ This section should take the full width of the page but yours has some white space by the side. Same with the `Partners` section

Original Design
-----
![image](https://user-images.githubusercontent.com/16058470/128344505-8f4a75b4-5a41-4be0-aa77-4b83face24cd.png)

Reply:
This issue is well resolved! ✔️